person.rb

```diff
@@ -1,14 +1,17 @@
 require './nameable'
+require './book'
+require './rental'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] It would be better if we set the getter and setter for the rentals. Here is a [link](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor`) to here us in fixing it

Reply:
@addod19  applied suggested changes 👍🏽